crates/core_simd/examples/matrix_inversion.rs

```diff
@@ -0,0 +1,319 @@
+//! 4x4 matrix inverse
+// Code ported from the `packed_simd` crate
+// Run this code with `cargo test --example matrix_inversion`
+#![feature(custom_inner_attributes)]
+#![deny(rust_2018_idioms)]
+#![allow(clippy::must_use_candidate)]
+#![feature(array_chunks)]
```

Comment:
Ctrl-F can't actually find any call to this?

Reply:
Oh neat, I only need the `array_chunks`. Thanks.